ecosystem/sep-0030.md

```diff
@@ -51,7 +51,11 @@ The registration flow is as follows:
 - The server responds to the client with the signing public key.
 - The client adds the signing public key as a signer of the account with an appropriate weight.
 
-A client can register with multiple servers and give each signing key a weight that is appropriate for the use case the client and servers are targeting. As an example, if a client is intending to use two servers that should individually have no control of the account but together are able to approve transactions, then the client can configure [account thresholds] to high 20, med 20, low 20, and assign weights 10 to both server signing keys.
+A client can register with multiple servers and give each signing key a weight that is appropriate for the use case the client and servers are targeting.
+
+As examples:
+- If a client intends to use two servers that should individually have no control of the account but together are able to approve transactions, then the client could configure [account thresholds] to high 2, med 2, low 2, and assign weights 1 to both server signing keys.
```

Comment:
I wouldn't mention medium or low threshold values here. This SEP only cares about the high threshold right? Theres no reason each threshold should be equal right? And in fact, they really shouldn't or wouldn't be IRL.

Reply:
I see your point that high should be irrelevant because for the SEP-10 proof only the high threshold is relevant, but this specific paragraph is discussing as an example how to configured account thresholds such that neither server has control of the account on their own. It is important to configure the medium and low thresholds greater than the servers signing weight otherwise the server would have some independent control. This is a minor point I think and so there isn't much discussion in the document on this. Ultimately the SEP provides an interface to support a wallet/server doing whatever they need for their product and doesn't go deep into prescribing the signer setups and only provides examples.